services/github-service/src/main/java/org/rhd/katapult/github/GithubResource.java

```diff
@@ -108,51 +172,52 @@ private JsonObject postToken(String code, String state, Client client)
       map.putSingle("state", state);
       // {"access_token":"3d4bf6b3ea93fba1dbfeeb5fa5afb5226e0cbec9","token_type":"bearer","scope":"public_repo,user:email"}
       JsonObject obj = client
-               .target("https://github.com/login/oauth/access_token")
-               .request()
-               .accept(MediaType.APPLICATION_JSON_TYPE)
-               .post(Entity.form(map), JsonObject.class);
+              .target("https://github.com/login/oauth/access_token")
+              .request()
+              .accept(MediaType.APPLICATION_JSON_TYPE)
+              .post(Entity.form(map), JsonObject.class);
       return obj;
    }
 
-   JsonObject forkRepository(String accessToken, String repository, Client client)
+   private JsonObject forkRepository(String accessToken, String repository, Client client)
    {
       Response response = client
-               .target("https://api.github.com/repos/" + repository + "/forks")
-               .request()
-               .accept(MediaType.APPLICATION_JSON_TYPE)
-               .header("Authorization", "token " + accessToken)
-               .header("User-Agent", "Github Forker")
-               .post(Entity.text(""));
-      int status = response.getStatus();
-      if (status == 202)
+              .target("https://api.github.com/repos/" + repository + "/forks")
+              .request()
+              .accept(MediaType.APPLICATION_JSON_TYPE)
+              .header("Authorization", "token " + accessToken)
+              .header("User-Agent", "Github Forker")
+              .post(Entity.text(""));
+      Response.StatusType status = response.getStatusInfo();
+      if (status == Response.Status.ACCEPTED)
       {
          return response.readEntity(JsonObject.class);
       }
       else
       {
          String msg = response.readEntity(String.class);
-         throw new WebApplicationException(msg, status);
+         log.warning("Failed to fork repository: " + repository + ", msg: "+ msg);
+         throw new WebApplicationException(msg, status.getStatusCode());
       }
    }
 
-   void createRepository(String accessToken, String repository, Client client)
+   private void createRepository(String accessToken, String repository, Client client)
    {
       JsonObjectBuilder json = Json.createObjectBuilder();
       json.add("name", repository)
-               .add("description", "Created via Forge Online")
-               .add("homepage", "http://forge.jboss.org")
-               .add("private", false)
-               .add("has_issues", true)
-               .add("has_wiki", true)
-               .add("has_downloads", true);
+              .add("description", "Created via Forge Online")
+              .add("homepage", "http://forge.jboss.org")
```

Comment:
Ok, this should be the name of the final github developer app so I'll factor that out into a constant or injected resource.